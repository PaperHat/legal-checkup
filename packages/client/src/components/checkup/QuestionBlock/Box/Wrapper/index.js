import styled from 'styled-components'

const Wrapper = styled.div`
  width: 658px;
  height:76px;
  font-family: MuseoSans;
  background-color:white;
  font-size: 32px;
  font-weight: 500;
  font-style: normal;
  font-stretch: normal;
  line-height: normal;
  letter-spacing: normal;
  text-align: center;
  margin: 0px auto;
  padding:58px 0 60px;
`
Wrapper.displayName = 'Wrapper'

export default Wrapper
