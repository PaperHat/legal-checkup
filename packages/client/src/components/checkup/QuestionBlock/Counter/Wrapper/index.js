import styled from 'styled-components'
import { DESKTOP, MOBILE, TABLET } from '../../../../../constants'
import styledMap from 'styled-map'

const Wrapper = styled.div`
  color: #b62645;
  width: 36px;
  height: 17px;
  font-family: MuseoSansCyrl;
  font-size: 13px;
  font-weight: 900;
  font-stretch: normal;
  line-height: normal;
  letter-spacing: normal;
  text-align: center;
  margin: 0px auto;
  padding-top: 18px;
  ${styledMap('format', {
    [DESKTOP]: `
   diplay:none;
    `,
    [MOBILE]: `
    
   `,
    [TABLET]: `
  display:none;
    `
  })}
`
Wrapper.displayName = 'Wrapper'

export default Wrapper
