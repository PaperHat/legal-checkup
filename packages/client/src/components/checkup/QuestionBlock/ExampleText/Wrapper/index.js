import styled from 'styled-components'
import { DESKTOP, MOBILE, TABLET } from '../../../../../constants'
import styledMap from 'styled-map'

const Wrapper = styled.div`
  width: 738px;
  background-color: #ffffff;
  margin: 0 auto;
  font-family: Helvetica;
  font-size: 16px;
  text-align: center;
  color: #9b9b9b;
  box-shadow: 0 3px 3px 0 rgba(0, 0, 0, 0.2);
  ${styledMap('format', {
    [DESKTOP]: `
    width: 738px;`,
    [MOBILE]: `
    width: 325px;;`,
    [TABLET]: `
    width: 708px;`
  })}
`
Wrapper.displayName = 'Wrapper'

export default Wrapper
