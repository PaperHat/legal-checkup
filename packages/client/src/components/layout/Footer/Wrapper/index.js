import styled from 'styled-components'
import styledMap from 'styled-map'

import { DESKTOP, MOBILE, TABLET } from '../../../../constants'

const Footer = styled.div`
  background-color: #202020;
  border-top: 7px solid #444141;
  display: flex;
  justify-content: center;
  align-items: center;

  ${styledMap('format', {
    [DESKTOP]: `
      height: 98px;`,
    [MOBILE]: `
      height: 135px;
      flex-direction: column;`,
    [TABLET]: `
      height: 100px;`
  })}
`
Footer.displayName = 'Footer'

export default Footer
