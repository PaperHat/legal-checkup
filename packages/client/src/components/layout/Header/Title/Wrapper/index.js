import styled from 'styled-components'
import styledMap from 'styled-map'

import { DESKTOP, TABLET, MOBILE } from '../../../../../constants'

const Wrapper = styled.div`
  font-family: 'Museo Sans Cyrl', sans-serif;
  font-weight: 900;
  line-height: 1;

  ${styledMap('format', { [`${DESKTOP}`]: `
      margin-left: 10px;
      font-size: 36px;
    `,
  [`${TABLET}`]: `
      margin-left: 10px;
      font-size: 25px;
    `,
  [`${MOBILE}`]: `
      margin-left: 8px;
      font-size: 18px;
    ` })};
`
Wrapper.displayName = 'Wrapper'

export default Wrapper
