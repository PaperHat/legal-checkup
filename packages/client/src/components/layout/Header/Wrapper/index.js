import styled from 'styled-components'
import styledMap from 'styled-map'

import {
  DESKTOP,
  MOBILE,
  TABLET,
  BLACK,
  GREYISH_BROWN
} from '../../../../constants'

const Wrapper = styled.header`
  background-color: ${BLACK};
  display: flex;
  width: 100%;

  ${styledMap('format', {
    [`${DESKTOP}`]: `
      height: 100px;
      flex-direction: row;
      align-items: center;
      border-bottom: 7px solid ${GREYISH_BROWN};
    `,
    [`${TABLET}`]: `
      height: 85px;
      flex-direction: row;
      border-bottom: 7px solid ${GREYISH_BROWN};
      align-items: center;
    `,
    [`${MOBILE}`]: `
      flex-direction: column;
    `
  })};
`
Wrapper.displayName = 'Wrapper'

export default Wrapper
