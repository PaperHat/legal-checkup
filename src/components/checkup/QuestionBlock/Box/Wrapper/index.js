import styled from 'styled-components'
import { DESKTOP, MOBILE, TABLET } from '../../../../../constants'
import styledMap from 'styled-map'

const Wrapper = styled.div`
  font-family: 'Museo Sans Cyrl 500', sans-serif;
  background-color: #ffffff;
  font-weight: 500;
  font-style: normal;
  font-stretch: normal;
  letter-spacing: normal;
  text-align: center;
  margin: 0px auto;
  ${styledMap('format', {
    [DESKTOP]: `
  font-size: 32px;
  line-height: normal;
  padding-top: 58px;
  padding-left: 40px;
  padding-right: 40px;`,
    [MOBILE]: `
  font-size: 25px;
  line-height: 1.2;
  padding-top: 12px;
  padding-left: 20px;
  padding-right: 20px;`,
    [TABLET]: `
  width: 630px;
  font-size: 32px;
  line-height: normal;
  padding-top: 58px;
  padding-left: 40px;
  padding-right: 40px;`
  })}
`
Wrapper.displayName = 'Wrapper'

export default Wrapper
