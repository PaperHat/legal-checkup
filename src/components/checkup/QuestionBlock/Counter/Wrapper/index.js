import styled from 'styled-components'

const Wrapper = styled.div`
  color: #b62645;
  width: 50px;
  height: 17px;
  font-family: 'Museo Sans Cyrl 900', sans-serif;
  font-size: 13px;
  font-weight: 900;
  font-stretch: normal;
  line-height: normal;
  letter-spacing: normal;
  text-align: center;
  margin: 0px auto;
  padding-top: 18px;
`
Wrapper.displayName = 'Wrapper'

export default Wrapper
