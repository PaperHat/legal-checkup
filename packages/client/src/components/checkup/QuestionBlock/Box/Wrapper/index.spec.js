import * as React from 'react'
import { create } from 'react-test-renderer'

import Wrapper from './'

describe('components:checkup:QuestionBlock:Box:Wrapper', function () {
  it('matches the snapshot', () => {
    expect(create(<Wrapper>Wrapper</Wrapper>).toJSON()).toMatchSnapshot()
  })
})
