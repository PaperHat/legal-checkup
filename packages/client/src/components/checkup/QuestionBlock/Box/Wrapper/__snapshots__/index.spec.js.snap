// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:checkup:QuestionBlock:Box:Wrapper matches the snapshot 1`] = `
.c0 {
  font-family: MuseoSans;
  background-color: #ffffff;
  font-weight: 500;
  font-style: normal;
  font-stretch: normal;
  -webkit-letter-spacing: normal;
  -moz-letter-spacing: normal;
  -ms-letter-spacing: normal;
  letter-spacing: normal;
  text-align: center;
  margin: 0px auto;
  font-size: 32px;
  line-height: normal;
  padding-top: 58px;
  padding-left: 40px;
  padding-right: 40px;
}

<div
  className="c0"
>
  Wrapper
</div>
`;
